src/AutoUI/index.tsx

```diff
@@ -99,11 +102,20 @@ export interface AutoUIProps<T> extends Omit<BoxProps, 'onChange'> {
 	) => void;
 	// TODO: onChange should also be called when data in the table is sorted and when columns change
 	/** Function that gets called when filters change */
-	onChange?: (changes: { filters?: JSONSchema[] }) => void;
+	onChange?: (changes: {
+		filters?: JSONSchema[];
+		oData?: { $filter?: any; $orderby?: any } | null;
+	}) => void;
+	/** Event emitted on page change */
+	onPageChange?: (page: number, itemsPerPage: number) => void;
```

Comment:
How about making these an extra property of the `onChange`?
In that case the `oData` could also include the $top, $skip, right away so that consumers don't have to repeat it in every page.
We are going to need to combine all those props whenever any one of them changes anyway, right?
```suggestion
	onChange?: (changes: {
		filters?: JSONSchema[];
		page: number,
		itemsPerPage?: number
		oData?: {
			$top?: number;
			$skip?: number;
			$filter?: any;
			$orderby?: any;
		} | null;
	}) => void;
```